internal/eval/eval.go

```diff
@@ -4232,3 +4232,49 @@ func selectRandomFields(hashMap HashMap, count int, withValues bool) []byte {
 
 	return clientio.Encode(results, false)
 }
+
+func evalAPPEND(args []string, store *dstore.Store) []byte {
```

Comment:
Ensure that the error messages match those of redis for all the error cases. Please also add tests for them.